app/AttributeTypes/EpochAttribute.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App\AttributeTypes;
+
+use App\Attribute;
+use App\ThConcept;
+use App\Exceptions\InvalidDataException;
+
+class EpochAttribute extends AttributeBase
+{
+    protected static string $type = "epoch";
+    protected static bool $inTable = false;
+    protected static ?string $field = 'json_val';
+    protected static bool $hasSelection = true;
+
+    public static function getSelection(Attribute $a) {
+        return ThConcept::getChildren($a->thesaurus_root_url, $a->recursive);
+    }
+
+    public static function fromImport(string $data) : mixed {
+        $startLabel = 'ad';
+        $endLabel = 'ad';
+        $parts = explode(';', $data);
+
+        if(count($parts) != 3) {
+            throw new InvalidDataException("Given data does not match this datatype's format (START;END;CONCEPT)");
+        }
+
+        $start = intval(trim($parts[0]));
+        $end = intval(trim($parts[1]));
+
+        if($end < $start) {
+            throw new InvalidDataException("Start date must not be after end data ($start, $end)");
+        }
```

Comment:
Basically, yes. But since they have different count of parts and additional code, I don't know if makes sense to store the shared code into a separate function or keep this basic code as it is.